wsdl/ver10/device/wsdl/devicemgmt.wsdl

```diff
@@ -379,6 +384,16 @@ IN NO EVENT WILL THE CORPORATION OR ITS MEMBERS OR THEIR AFFILIATES BE LIABLE FO
     			</xs:restriction>
 			</xs:simpleType>
 			<!--===============================-->
+			<xs:simpleType name="Addons">
+				<xs:restriction base="xs:string">
+					<xs:enumeration value="TLSConfiguration">
+						<xs:annotation>	
+							<xs:documentation>Device supports the TLS Configuration Adoon.</xs:documentation>
```

Comment:
Typo correction in annotation 
Adoon--> Addon.

Reply:
Fixed!